### ISSUE TITLE
Dynamic Trees Quark 1.20.1 Compat Issue With Twighlight Forest

### ISSUE THREAD

**Issue by Franksy32** [DynamicTreesTeam/DynamicTrees-Quark#13]
With DT quark is installed alongside twilight forest in 1.20.1 it seems to prevent the portal opening to enter the twilight forest.

**Comment by copygirl** [comment]
That could be related to this mod breaking a bunch of tags, which in my case also leads to axes not working as expected?
```
[02Jul2024 17:40:30.662] [Worker-Main-13/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag dynamictrees:seeds as it is missing following references: 
	dtquark:sweet_blossom_seed (from DynamicTreesQuark-1.20.1-2.5.0.jar)
[02Jul2024 17:40:30.674] [Worker-Main-11/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag dynamictrees:leaves as it is missing following references: 
	dtquark:sweet_blossom_leaves (from DynamicTreesQuark-1.20.1-2.5.0.jar)
[02Jul2024 17:40:30.681] [Worker-Main-11/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag dynamictrees:saplings as it is missing following references: 
	dtquark:sweet_blossom_sapling (from DynamicTreesQuark-1.20.1-2.5.0.jar)
```

**Comment by copygirl** [comment]
This appears to be because the sweet blossom / trumpet tree was removed from Quark in 1.20.x due to the addition of cherry blossom trees to vanilla Minecraft. This should be fixed by simply removing the related tag entries that are referring to items that no longer exist.

The reason this breaks Twilight Forest portals might be due of the cascading effect of tag loading failures, that end up breaking `minecraft:saplings` tag which then breaks the tag that Twilight Forest uses to check for valid plant blocks that can form its portal.

**Comment by copygirl** [comment]
I've just confirmed that the mod "Load My F***ing Tags" ([Modrinth](https://modrinth.com/mod/lmft) / [CurseForge](https://www.curseforge.com/minecraft/mc-mods/lmft)) provides a workaround for this bug.

**Comment by supermassimo** [comment]
should be fixed on 2.5.1

**Comment by copygirl** [comment]
Can confirm. No more tag breakage.
